src/api/todos.ts

```diff
@@ -0,0 +1,18 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+export const getTodos = (userId: number) => {
+  return client.get<Todo[]>(`/todos?userId=${userId}`);
+};
+
+export const createTodos = (userId: number, todo: Todo) => {
```

Comment:
```suggestion
export const createTodo = (userId: number, todo: Todo) => {
```
Actually, it creates only one todo